cloudinit/config/schema.py

```diff
@@ -843,32 +879,36 @@ def _get_property_description(prop_config: dict) -> str:
     Order and deprecated property description after active descriptions.
     Add a trailing stop "." to any description not ending with ":".
     """
-    prop_descr = prop_config.get("description", "")
+
+    def assign_descriptions(
+        config: dict, descriptions: list, deprecated_descriptions: list
+    ):
+        if any(
+            [
```

Comment:
Nit: what do you think about the following change?

```diff
diff --git a/cloudinit/config/schema.py b/cloudinit/config/schema.py
index ffc3dfe91..acab72806 100644
--- a/cloudinit/config/schema.py
+++ b/cloudinit/config/schema.py
@@ -884,11 +884,10 @@ def _get_property_description(prop_config: dict) -> str:
         config: dict, descriptions: list, deprecated_descriptions: list
     ):
         if any(
-            [
-                config.get("deprecated_version"),
-                config.get("changed_version"),
-                config.get("new_version"),
-            ]
+            map(
+                config.get,
+                ("deprecated_version", "changed_version", "new_version"),
+            )
         ):
             deprecated_descriptions.append(
                 _add_deprecated_changed_or_new_msg(config)
```

Reply:
I like it